lib/collection.js

```diff
@@ -2079,6 +2102,8 @@ Collection.prototype.initializeUnorderedBulkOp = function(options) {
  *
  * @method
  * @param {object} [options] Optional settings.
+ * @param {boolean} [options.bypassDocumentValidation=false] Allow driver to bypass schema validation in MongoDB 3.2 or higher.
+ * @param {boolean} [options.checkKeys=true] Enforce document key checking.
```

Comment:
are `checkKeys` and `bypassDocumentValidation` bson options?

Reply:
`checkKeys` is so I'll remove that. `bypassDocumentValidation` is not.